### PR TITLE
DOP-4761: force render all tabs (and fix SVG icons)

### PR DESCRIPTION
**Pull request by seungpark**
### Stories/Links:

DOP-4761

### Current Behavior:

[Manual page with tabbed content](https://www.mongodb.com/docs/manual/tutorial/update-documents/)
[Staging link from previous PR shows icons not working correctly](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/seung.park/DOP-4761/driver-connection/index.html) 

### Staging Links:

[Manual page with tabbed content](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/docs/seung.park/DOP-4761-address-icons/tutorial/update-documents/index.html)
[Staging link of previous problematic page shows icons working correctly](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/seung.park/DOP-4761-address-icons/driver-connection/index.html)

### Notes:
- Seems like creating a unique hash for each icon, and appending this unique hash to the `<LinearGradient>`s id solves this problem. Most likely that there was a clash of id's when using multiple of the same SVG

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [X] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: src/components/Tabs/index.js [hunk 1]
```diff
@@ -152,10 +152,12 @@ const Tabs = ({ nodeData: { children, options = {} }, page, ...rest }) => {
                 : tabId;
 
             return (
-              <LeafyTab className={cx(getTabStyling({ isProductLanding }))} key={tabId} name={tabTitle}>
-                {tab.children.map((child, i) => (
-                  <ComponentFactory {...rest} key={`${tabId}-${i}`} nodeData={child} />
-                ))}
+              <LeafyTab className={'test-leafy-tab-here'} key={tabId} name={tabTitle}>
```
Comment (rayangler) [inline src/components/Tabs/index.js:155]: Do we still need the class name?

File: src/components/icons/Ruby.js [hunk 2]
```diff
@@ -1,154 +1,172 @@
...
+        d="M82.113 156.507l39.313 75.848c23.246-12.607 41.45-27.967 58.121-44.42l-97.434-31.428z"
+        fill={`url(#${hash}n)`}
+      />
+      <path
+        d="M17.174 171.34l-5.57 66.328c10.51 14.357 24.97 15.605 40.136 14.486-10.973-27.311-32.894-81.92-34.566-80.814z"
+        fill={`url(#${hash}o)`}
+      />
+      <path
+        d="M174.826 22.654l78.1 10.96c-4.169-17.662-16.969-29.06-38.787-32.623l-39.313 21.663z"
+        fill={`url(#${hash}p`}
```
Comment (rayangler) [inline src/components/icons/Ruby.js:166]: Missing closing parenthesis here

File: src/components/icons/Ruby.js [hunk 3]
```diff
@@ -1,154 +1,172 @@
...
+        </radialGradient>
+      </defs>
+      <path d="M197.467 167.764l-145.52 86.41 188.422-12.787L254.88 51.393l-57.414 116.37z" fill={`url(#${hash}a)`} />
+      <path d="M240.677 241.257L224.482 129.48l-44.113 58.25 60.308 53.528z" fill={`url(#${hash}b)`} />
+      <path d="M240.896 241.257l-118.646-9.313-69.674 21.986 188.32-12.673z" fill={`url(#${hash}c)`} />
+      <path d="M52.744 253.955l29.64-97.1L17.16 170.8l35.583 83.154z" fill={`url(#${hash}d)`} />
+      <path d="M180.358 188.05L153.085 81.226l-78.047 73.16 105.32 33.666z" fill={`url(#${hash}e)`} />
+      <path d="M248.693 82.73l-73.777-60.256-20.544 66.418 94.321-6.162z" fill={`url(#${hash}f)`} />
+      <path d="M214.191.99L170.8 24.97 143.424.669l70.767.322z" fill={`url(#${hash}g)`} />
+      <path d="M0 203.372l18.177-33.151-14.704-39.494L0 203.372z" fill={`url(#${hash}h`} />
```
Comment (rayangler) [inline src/components/icons/Ruby.js:130]: Missing closing parenthesis here

**Review by rayangler — COMMENTED** [review]
Thanks for looking into this and finding this solution. I pointed out a few things we might want to address before formally merging, but otherwise lgtm.

File: src/components/Tabs/index.js (hunk 1 above)
Comment (seungpark) [reply src/components/Tabs/index.js:155]: good catch! removed

**Review by rayangler — APPROVED** [review]

**Review by mmeigs — APPROVED** [review]
This is fantastic. Thank you for figuring this one out!! 

Sidenote: I have never noticed us using Perl as a language, and that it doesn't have an icon. I'm guessing we should drop in a ticket to fix that/ask design?

**Comment by seungpark** [comment]
> This is fantastic. Thank you for figuring this one out!!
> 
> Sidenote: I have never noticed us using Perl as a language, and that it doesn't have an icon. I'm guessing we should drop in a ticket to fix that/ask design?

i think some icons don't exist, aka Motor, Perl - will drop in a design ticket for this!

**Comment by seungpark** [comment]
> > This is fantastic. Thank you for figuring this one out!!
> > Sidenote: I have never noticed us using Perl as a language, and that it doesn't have an icon. I'm guessing we should drop in a ticket to fix that/ask design?
> 
> i think some icons don't exist, aka Motor, Perl - will drop in a design ticket for this!

@mmeigs https://jira.mongodb.org/browse/EXPO-3904 !

**Pull request merged**